### DISCUSSION TITLE
Where is the dll file?

### DISCUSSION THREAD

**Discussion by dwsiddall** [YimMenu/HorseMenu#97] [General]
I feel like an idiot, but when I look at list of files (under the <>Code tab), I don't see the dll.  And when I download the zipped package, there is no dll.

What am I missing?

**Comment by Rxann** [comment]
Go to the releases tab, and then it will be there. it is not included in the actual code section. it will be off to the middle right when you first come to the repo page.

**Reply by dwsiddall** [reply]
Thank you!

**Reply by Rxann** [reply]
your welcome

**Comment by dwsiddall** [comment]
After downloading the latest DLL, I am noticing that I can no longer set actions to the F-Keys.  The previous DLL, for example, I could set Teleport to Waypoint to F5.  With this newest DLL I cannot.

EDIT:  After unloading the MOD, exiting the game, then re-starting the game and re-injecting the Mod, then the F-Key assignment worked, but only for a few minutes.  After that, I can longer assign an action to the F-Keys.

Then the Teleport stopped working with a message about failing to get ground coordinates.

Is there a way to get the previous DLL until this is sorted?

[Forgive me if I am putting this in the wrong place...]

**Comment by Rxann** [comment]
> After downloading the latest DLL, I am noticing that I can no longer set actions to the F-Keys.  The previous DLL, for example, I could set Teleport to Waypoint to F5.  With this newest DLL I cannot.
> 
> 
> 
> EDIT:  After unloading the MOD, exiting the game, then re-starting the game and re-injecting the Mod, then the F-Key assignment worked, but only for a few minutes.  After that, I can longer assign an action to the F-Keys.
> 
> 
> 
> Then the Teleport stopped working with a message about failing to get ground coordinates.
> 
> 
> 
> Is there a way to get the previous DLL until this is sorted?
> 
> 
> 
> [Forgive me if I am putting this in the wrong place...]

Hmm, this is interesting. Hot keys are kind of broken right now. Normally for feature requests and bug reports you should go to the issue tab.

**Comment by dwsiddall** [comment]
Mainly I was wondering if I could get the previous DLL.  It worked fine.  Is there a file history section?

**Reply by Rxann** [reply]
Yes, in the same place you got the current one, go there and click releases. it should show you the prior one.

**Reply by dwsiddall** [reply]
So sorry, but when I go to "Releases" I only see the current Release (see attached photo).

If there's something there I am not seeing...I apologize.


![Image1](https://github.com/YimMenu/HorseMenu/assets/30631241/6f327f9d-bd25-4208-bf45-dc2bab286d8e)

**Reply by Rxann** [reply]
Oh, I forgot this repo does it like this. Sorry, there is no old binary you can use.
